File: src/main/java/frc/robot/commands/ToggleIntakeMotors.java
// Copyright (c) FIRST and other WPILib contributors.
// Open Source Software; you can modify and/or share it under the terms of
// the WPILib BSD license file in the root directory of this project.

package frc.robot.commands;

import edu.wpi.first.wpilibj2.command.CommandBase;
import frc.robot.subsystems.IntakeSubsystem;

/**
 * An example command that uses an example subsystem.
 */
public class ToggleIntakeMotors extends CommandBase {

  private final IntakeSubsystem intakeSubsystem;  

  /**
   * bring the intake in or out
   * */
  public ToggleIntakeMotors(IntakeSubsystem intake) {
    intakeSubsystem = intake;
    addRequirements(intakeSubsystem);
  }

  @Override
  public void initialize() {
    intakeSubsystem.toggleIntakeMotors();
  }

  // Returns true when the command should end.
  @Override
  public boolean isFinished() {
    return true;
  }
}

File: src/main/java/frc/robot/commands/ToggleIntakeMotors.kt
// Copyright (c) FIRST and other WPILib contributors.
// Open Source Software; you can modify and/or share it under the terms of
// the WPILib BSD license file in the root directory of this project.
package frc.robot.commands

import edu.wpi.first.wpilibj2.command.CommandBase
import frc.robot.subsystems.IntakeSubsystem

/**
 * An example command that uses an example subsystem.
 */
class ToggleIntakeMotors(private val intakeSubsystem: IntakeSubsystem) : CommandBase() {
    /**
     * bring the intake in or out
     */
    init {
        addRequirements(intakeSubsystem)
    }

    override fun initialize() {
        intakeSubsystem.toggleIntakeMotors()
    }

    // Returns true when the command should end.
    override fun isFinished(): Boolean {
        return true
    }
}
